platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -74,6 +75,22 @@ or
 
 > Calculating task graph as configuration cache cannot be reused because file '.../some-file.txt' **has been replaced by a directory**.
 
+<a name="ip-usability"></a>
+
+### Isolated Projects usability improvements
+
+#### String-notated task dependency is not a violation anymore
+
+Depending on a task from another project in string-notated form is a common idiom:
+
+```
+foo.dependsOn(":a:bar")
+```
+
+Starting with this release, this is no longer considered a violation of Isolated Projects boundaries.
+
+#### `gradle init` generates Isolated Projects compatible projects
+[Build Init Plugin](userguide/build_init_plugin.html) supports multimodule project layout. Starting with this release, `gradle init` generates projects compatible with Isolated Projects restrictions.
```

Comment:
```suggestion
The [Build Init Plugin](userguide/build_init_plugin.html) supports creating multi-module projects. 
Starting with this release, `gradle init` also generates projects compatible with Isolated Projects restrictions.
```

Reply:
@lkasso This `also` word is a bit confusing IMO. There is no option to generate project either compatible with IP or not. Should we omit it?